CRS/Cirrus.Node/NonFungibleTokenLogEnrichment.cs

```diff
@@ -0,0 +1,58 @@
+using System.Collections.Generic;
+using System.Text;
+using NBitcoin;
+using Stratis.Bitcoin.Features.SmartContracts.Models;
+using Stratis.SmartContracts;
+using Stratis.SmartContracts.CLR;
+using Stratis.SmartContracts.Core.Receipts;
+using Stratis.SmartContracts.Core.State;
+
+namespace Cirrus.Node
+{
+    public class NonFungibleTokenLogEnrichment : ISmartContractLogEnrichment
+    {
+        private readonly IStateRepositoryRoot stateRoot;
+        private readonly Network network;
+        private readonly ISerializer serializer;
+
+        public NonFungibleTokenLogEnrichment(IStateRepositoryRoot stateRoot, Network network)
+        {
+            this.stateRoot = stateRoot;
+            this.network = network;
+        }
+
+        const string typeName = "NonFungibleToken";
+
+        public bool IsSmartContractSupported(string typeName)
+        {
+            return typeName == NonFungibleTokenLogEnrichment.typeName;
+        }
+
+        public void EnrichLogs(Receipt receipt, List<LogResponse> logResponses)
+        {
```

Comment:
This only enriches the constructor trx, you might want to mae logs on other methods will they go here or will they get a new class?